src/main/java/com/acme/dbo/txlog/ConsoleOutput.java

```diff
@@ -0,0 +1,7 @@
+package com.acme.dbo.txlog;
+
+public class ConsoleOutput {
```

Comment:
Why separate class for this responsibility?

Reply:
I think it's the right way - we separate decorator logic to it's own class